include/openthread/thread.h

```diff
@@ -466,6 +466,38 @@ otError otThreadSetMeshLocalPrefix(otInstance *aInstance, const otMeshLocalPrefi
  */
 const otIp6Address *otThreadGetLinkLocalIp6Address(otInstance *aInstance);
 
+/**
+ * This function returns the Thread Link-Local All Thread Nodes multicast address.
+ *
+ * The address is a link-local Unicast Prefix-Based Multcast Address [RFC 3306], with:
+ *   - flgs set to 3 (P = 1 and T = 1)
+ *   - scop set to 2
+ *   - plen set to 64
+ *   - group ID set to 1
```

Comment:
Should we also indicate that the "network prefix" is set to the Mesh Local Prefix?

Reply:
Updated.